### PR TITLE
Fix  mismatch between SV n.genes in case report and SV variants page

### LINKED ISSUE #3285: Different number of genes in SV variantS page and general report
Take for instance the first SV variant of this case, an inversion that contains 197 genes: https://scout-stage.scilifelab.se/cust002/F0042222/sv/variants?variant_type=clinical

I marked it as causative, so it shows on the general report

But when you see the report it says that it has 32 genes!

### PR DESCRIPTION
**Pull request by northwestwitch**
Tentative fix #3285 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested with same variant as in the open issue

Case report using this branch: 🆗 

image


Same variant, dismissed: 🆗 

image

File: scout/constants/variant_tags.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "genetic_models",
     "acmg_classification",
     "genes",
+    "hgnc_symbols",
```
Comment (northwestwitch) [inline scout/constants/variant_tags.py:35]: I hate this change we introduced some time ago (https://github.com/Clinical-Genomics/scout/pull/2861). Every time something is missing from general report but should be there it takes me forever to figure out why! I wish we could revert 😠

Comment (dnil) [reply scout/constants/variant_tags.py:35]: Heh, yes, I guess the react version of the report is still on hold, so perhaps the use case isn't really there?

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 2]
```diff
@@ -791,7 +791,7 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
 {% macro dismissed_gene_list(variant) %}
   {% if variant.genes %}
     {% if variant.genes|length > 3%}
-      {{ variant.genes|length }}&nbsp;genes
+      {{ variant.hgnc_symbols|length }}&nbsp;genes
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:794]: Perphaps also add a small note in e.g. the controller or view docstring describing the difference so it's not confusing if one needs to change it later?

**Review by dnil — APPROVED** [review]
Well found!

File: scout/server/blueprints/cases/templates/cases/case_report.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/case_report.html:794]: Good idea, I'll fix!

**Pull request merged**